src/test/java/com/github/havardh/javaflow/model/ModelWithIgnoredField.java

```diff
@@ -0,0 +1,14 @@
+package com.github.havardh.javaflow.model;
+
+import com.github.havardh.javaflow.annotations.JsonIgnore;
+
+public class ModelWithIgnoredField {
+  @JsonIgnore
+  private String field;
+
+  @JsonIgnore
```

Comment:
I think usually the `@JsonIgnore` will only be present on either getter _or_ field, so maybe have 2 fields here to get both combinations?

Reply:
Hi @pascalgn, thanks for digging into this. :) 

On that note I guess I need to consider these annotations in more detail. 

The [`javadoc`](https://fasterxml.github.io/jackson-annotations/javadoc/2.5/com/fasterxml/jackson/annotation/JsonIgnore.html) talks about this case in particular, and goes further to describe what happens when you mix and match `JsonIgnore` and `JsonProperty`. I think I need to make some more explorations to make the same semantics for `javaflow` and `Jackson`.